packages/react-vapor/src/components/loading/components/TableLoading.tsx

```diff
@@ -2,36 +2,96 @@ import * as classNames from 'classnames';
 import * as React from 'react';
 import * as _ from 'underscore';
 
-const Table = ({numberOfColumns = 4, numberOfRow = 10}: {numberOfColumns?: number; numberOfRow?: number}) => {
+const Table = ({
+    isCard = false,
+    numberOfColumns = 4,
+    numberOfRow = 10,
+    numberOfSubRow = 3,
+}: {
+    isCard?: boolean;
+    numberOfColumns?: number;
+    numberOfRow?: number;
+    numberOfSubRow?: number;
+}) => {
     return (
         <>
             <table className="table big-table">
-                <Body numberOfColumns={numberOfColumns} numberOfRow={numberOfRow} />
+                <Body
+                    isCard={isCard}
+                    numberOfColumns={numberOfColumns}
+                    numberOfRow={numberOfRow}
+                    numberOfSubRow={numberOfSubRow}
+                />
             </table>
         </>
     );
 };
 
-const Body = ({numberOfColumns = 4, numberOfRow = 10}: {numberOfColumns?: number; numberOfRow?: number}) => (
-    <tbody>
-        {_.times(numberOfRow, (nColumn: number) => (
-            <tr key={`table-row-loading-${nColumn}`} className="mod-border-bottom no-hover">
-                {_.times(numberOfColumns, (nRow: number) => (
-                    <Row key={`table-row-loading-${nRow}`} num={nColumn} />
-                ))}
-            </tr>
-        ))}
-    </tbody>
-);
+const Body = ({
+    isCard = false,
+    numberOfColumns = 4,
+    numberOfRow = 10,
+    numberOfSubRow = 3,
+}: {
+    isCard?: boolean;
+    numberOfColumns?: number;
+    numberOfRow?: number;
+    numberOfSubRow?: number;
+}) => {
+    return (
+        <tbody>
+            {isCard
+                ? _.times(numberOfRow, (nColumn: number) => (
```

Comment:
is a ternary the best thing to use here :thinking:  ?

Reply:
isCard && would work, but did you have something else in mind?

Reply:
I'd probably extract the two components, then in <tbody> for instance:
`{_.times(numberOfRow, (nColumn: number) => { isCard ? (<CardLoading />) : <RowLoading /> })}`